contracts/senders/aave-v2-migrator/events.sol

```diff
@@ -5,25 +5,41 @@ contract Events {
     event LogDeposit(
         address owner,
         address[] tokens,
-        uint[] amts
+        uint256[] amts
     );
 
     event LogWithdraw(
         address owner,
         address[] tokens,
-        uint[] amts
+        uint256[] amts
+    );
```

Comment:
@mubaris @KaymasJain We can remove these, right?

Reply:
Yep!